### PR TITLE
fix broken links

### PR DESCRIPTION
**Pull request by JounQin**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

Fix broken links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -268,6 +268,6 @@ We welcome your contributions! Please have a read of
 [chat]: https://badges.gitter.im/webpack/webpack.svg
 [chat-url]: https://gitter.im/webpack/webpack
 
-[dev-ware]: /webpack/webpack-dev-middleware
-[hot-client]: /webpack-contrib/webpack-hot-client
+[dev-ware]: https://github.com/webpack/webpack-dev-middleware#options
+[hot-client]: https://github.com/webpack/webpack-dev-middleware#options
```
Comment (shellscape) [inline README.md]: not sure why you changed this one

**Comment by shellscape** [comment]
I don't understand the motivation for the change. The links work as expected in the README.

File: README.md (hunk 1 above)
Comment (JounQin) [reply README.md]: Really? I open these links on GitHub and they are https://github.com/webpack-contrib/webpack-serve/blob/master/webpack/webpack-dev-middleware and https://github.com/webpack-contrib/webpack-serve/blob/master/webpack-contrib/webpack-hot-client.

**Comment by JounQin** [comment]
@shellscape Are you opening them on localhost rather than on GitHub actually?

File: README.md (hunk 1 above)
Comment (shellscape) [reply README.md]: no I mean you changed hot-client's link to point to webpack-dev-middleware. Line 272 doesn't look like you made the change you were going for there

Comment (JounQin) [reply README.md]: Sorry about that, fixed.

**Comment by shellscape** [comment]
Cheers, thanks.

**Pull request merged**